server/test/sc/server/plugins/TestGame.kt

```diff
@@ -0,0 +1,96 @@
+package sc.server.plugins
+
+import org.slf4j.LoggerFactory
+import sc.api.plugins.IGameState
+import sc.api.plugins.exceptions.TooManyPlayersException
+import sc.framework.plugins.ActionTimeout
+import sc.framework.plugins.Player
+import sc.framework.plugins.RoundBasedGameInstance
+import sc.protocol.responses.ProtocolMessage
+import sc.server.helpers.TestTeam
+import sc.shared.*
+import java.util.*
+
+class TestGame : RoundBasedGameInstance<TestPlayer?>(TestPlugin.TEST_PLUGIN_UUID) {
+    private val state = TestGameState()
+    override fun onRoundBasedAction(fromPlayer: Player, data: ProtocolMessage) {
+        /*
+         * NOTE: Checking if right player sent move was already done by
+         * {@link sc.framework.plugins.RoundBasedGameInstance#onAction(Player, Object)}.
+         * There is no need to do it here again.
+         */
+
+        if (data is TestMove) {
+            data.perform(state)
+            next(if (state.currentPlayer === TestTeam.RED) state.red else state.blue)
+        }
+    }
+
+    override fun checkWinCondition(): WinCondition? {
+        return if (this.round > 1) {
+            logger.info("Someone won")
```

Comment:
I don't think this log message is helpful

Reply:
71be7e58